### PR TITLE
ES6 cli

### PR DESCRIPTION
**Pull request by levithomason**
>**Heads up!**
>Merge #10 first, this PR includes changes from there.

This PR:

1. refactors `/src/cli.js` to ES6 syntax
1. extends `airbnb` (instead of legacy)

### CODE REVIEW

File: src/printCmdRet.js [hunk 1]
```diff
@@ -1,19 +1,17 @@
 // This function takes the raw result of a shelljs command and figures out how to print it.
 // Invoke this *REGARDLESS* of what the command returns, it will figure it out.
-function printCmdRet(ret) {
+export const printCmdRet = (ret) => {
```
Comment (ariporad) [inline src/printCmdRet.js]: This should be `export default`.

**Comment by ariporad** [comment]
One small nit.

File: src/printCmdRet.js (hunk 1 above)
Comment (levithomason) [inline src/printCmdRet.js]: whoops, thanks!

**Comment by levithomason** [comment]
Since we're dropping arg parse for now, I'm gonna close this PR and make all es6 updates in #10 only.

**Pull request closed without merging**